build.gradle

```diff
@@ -70,6 +70,7 @@ dependencies {
     implementation group: 'org.apache.commons', name: 'commons-text', version: '1.6'
     implementation group: 'net.dankito.readability4j', name: 'readability4j', version: '1.0.3'
     implementation group: 'net.sourceforge.cssparser', name: 'cssparser', version: '0.9.27'
+    implementation group: 'org.asynchttpclient', name: 'async-http-client', version: '2.4.3'
```

Comment:
Should we use the latest 2.8.1 version instead? https://mvnrepository.com/artifact/org.asynchttpclient/async-http-client

Reply:
Done